src/main/groovy/wooga/gradle/unity/UnityTask.groovy

```diff
@@ -63,14 +64,15 @@ abstract class UnityTask extends DefaultTask
 
                 def unityPath = unityPath.get().asFile.absolutePath
                 def unityArgs = getAllArguments()
-
-                OutputStream outputStream = getOutputStream()
+                def unityEnvironment = environment.get()
```

Comment:
As I mentioned. The `MapProperty` class implements the `Map` interface. I don't think it is needed to fetch it explicitly. Or does it throw fits?

Reply:
The exec call's doesn't is of Map type (no property) so I had to pass the unwrapped value.

Reply:
![image](https://user-images.githubusercontent.com/10147802/122214061-f29b7000-cea9-11eb-9e6a-e87e4fc47479.png)

Reply:
ok then.